pkgs/development/tools/rust/cargo-deny/default.nix

```
@@ -2,39 +2,47 @@
 , rustPlatform
 , fetchFromGitHub
 , pkg-config
+, libgit2_1_5
 , openssl
+, zlib
 , zstd
 , stdenv
 , curl
-, Security
+, darwin
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-deny";
-  version = "0.13.7";
+  version = "0.13.8";
 
   src = fetchFromGitHub {
     owner = "EmbarkStudios";
     repo = pname;
     rev = version;
-    sha256 = "sha256-E9tFzac6WkEGfsXj1nykQAR20+5Pi5xMd82MeDed9qg=";
+    hash = "sha256-6RmC54DsL31/SBrA16p+iub7VTQvc1nG+ml+6Xaj0NM=";
   };
 
-  # enable pkg-config feature of zstd
-  cargoPatches = [ ./zstd-pkg-config.patch ];
-
-  cargoSha256 = "sha256-E3Gg7PwBNVkvX2vqtbCxz0kbe1ZWrcAWxj6OJtENBe0=";
+  cargoHash = "sha256-u3hv1phSwZ4oILFMWIAqWhdt0jI8ao3gje1WZA3ZFo8=";
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [ openssl zstd ]
-    ++ lib.optionals stdenv.isDarwin [ curl Security ];
+  buildInputs = [
+    libgit2_1_5
+    openssl
+    zlib
+    zstd
+  ] ++ lib.optionals stdenv.isDarwin [
+    curl
+    darwin.apple_sdk.frameworks.Security
+  ];
 
   buildNoDefaultFeatures = true;
 
   # tests require internet access
   doCheck = false;
 
+  ZSTD_SYS_USE_PKG_CONFIG = true;
```

```suggestion
  env.ZSTD_SYS_USE_PKG_CONFIG = true;
```

I think these get prefixed now?

Also if this could be moved up towards `buildInputs` that'd be great IMO

---

I put it in a `env = { };` to make adding new env vars easier, I moved it up, but after `buildNoDefaultFeatures` since `buildNoDefaultFeatures` is closer to `buildFlags` IMO, which goes before `env` in [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md), and it also affects `buildInputs`